### PR TITLE
Resolve syncing conflicts from repo_sync_working_branch to live

### PR DESCRIPTION
**Pull request by openpublishbuild**
Pull request opened by Docs to resolve syncing conflicts from repo_sync_working_branch to live.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/meeting-settings-in-teams.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ If you're using Quality of Service [(QoS)](qos-in-teams.md) to prioritize networ
     ![Screenshot of the network settings for meetings in the admin center](media/meeting-settings-network.png "Screenshot of the network settings for Teams meetings in the Microsoft Teams admin center")
 
     - To allow DSCP markings to be used for QoS, turn on **Insert Quality of Service (QoS) markers for real-time media traffic**. You only have the option of using markers or not; you can't set custom markers for each traffic type. See [Select a QoS implementation method](QoS-in-Teams.md#select-a-qos-implementation-method) for more on DSCP markers.
-        > [!NOTE] 
+        > [!NOTE]
         > Turning on **Insert Quality of Service (QoS) markers for real-time media traffic** will also enable communication to the Transport Relay with UDP ports 3479 (Audio), 3480 (Video) and 3481 (Sharing).
```
Comment (JohanFreelancer9) [inline Teams/meeting-settings-in-teams.md:91]:
```suggestion
        > Turning on **Insert Quality of Service (QoS) markers for real-time media traffic** will also enable communication to the Transport Relay with UDP ports 3479 (Audio), 3480 (Video), and 3481 (Sharing).
```

File: Teams/teams-security-guide.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ appliesto:
 > The Teams service model is subject to change in order to improve customer experience. For example, the default access or refresh token expiration times may be subject to modification in order to improve performance and authentication resiliency for those using Teams. Any such changes would be made with the goal of keeping Teams secure and Trustworthy by Design.
 <p>
 
- # Security and Microsoft Teams
+# Security and Microsoft Teams
 
 Microsoft Teams, as part of the Microsoft 365 (M365) service, follows all the security best practices and procedures such as service-level security through defense-in-depth, customer controls within the service, security hardening and operational best practices. For full details, please see the [Microsoft Trust Center](https://microsoft.com/trustcenter).
```
Comment (JohanFreelancer9) [inline Teams/teams-security-guide.md:29]:
```suggestion
Microsoft Teams, as part of the Microsoft 365 (M365) service, follows all the security best practices and procedures such as service-level security through defense-in-depth, customer controls within the service, security hardening, and operational best practices. For full details, please see the [Microsoft Trust Center](https://microsoft.com/trustcenter).
```

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**